src/components/Player/Pages/Overview/CountsSummary.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import GaugeChart from './../../../Visualizations/GaugeChart';
+
+const Styled = styled.div`
+    justify-content: space-evenly;
+    display: flex;
+    flex-wrap: wrap;
+    border: 1px solid rgba(255,255,255,0.1);
+    border-radius: 10px;
+
+    > div {
+        margin-top: 3px;
+    }
+`;
+
+const Summary = ({ data }) => (
+  <Styled>
+    {data.map(el => <GaugeChart value={el.winPercent} caption={el.category} />)}
```

Comment:
It's not very clear these are winrates, they look more like proportions.  I wonder if there's a good way to show both?

Reply:
I changed the styling. Hopefully now it's more clear.